modules/eap-73-env/7.3.1/module.yaml

```diff
@@ -21,7 +21,7 @@ labels:
       description: "Environment variable used to specify the debug port. If not set, the default EAP debug port will be used (8787). Only applicable when development mode is enabled."
 envs:
     - name: "WILDFLY_VERSION"
-      value: "7.3.0.GA-redhat-SNAPSHOT"
+      value: "7.3.1.GA-redhat-00001"
```

Comment:
I think this is the final artifact name? But lets hold and make sure.

Reply:
@luck3y we will probably know tomorrow when QE give me a feedback if the current CR1 bits can be promoted to QE for full testing.